src/transformers/data/processors/utils.py

```diff
@@ -46,11 +46,42 @@ class InputExample:
     text_a: str
     text_b: Optional[str] = None
     label: Optional[str] = None
+    seq_length: int = None
```

Comment:
this is lacking a docstring and should be optional for backward compatibility - also is this required for FastFormers?

Reply:
Thanks @patrickvonplaten ! I will add a docstring and make it optional.